### DISCUSSION TITLE
[RESOLVED] "Format document" does not respect tab settings

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [github/codeql#8657] [General]
My tab settings is 4 spaces, but codeql does not respect that configuration. It always use "2 spaces" while formatting the document, e.g

![image](https://user-images.githubusercontent.com/1357701/161457309-b63c5010-635c-4e52-9780-932c668d8adf.jpg)

**Comment by smowton** [comment]
This is a deliberate design choice: CodeQL files always use 2-space indentation per https://github.com/github/codeql/blob/main/docs/ql-style-guide.md
